### PR TITLE
feat(cli): docker image asset scanning by default

### PR DESCRIPTION
**Pull request by jogold**
Amazon ECR image scanning helps in identifying software vulnerabilities in container images.

Amazon ECR uses the Common Vulnerabilities and Exposures (CVEs) database from the open source CoreOS Clair project and provides a list of scan findings.

Image Scanning for Amazon ECR is available at no additional charge, and you can now use it in all commercial AWS Regions and GovCloud (US)

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/aws-cdk/lib/api/toolkit-info.ts [hunk 1]
```diff
@@ -143,6 +143,14 @@ export class ToolkitInfo {
       lifecyclePolicyText: JSON.stringify(DEFAULT_REPO_LIFECYCLE)
     }).promise();
 
+    // Activate image scanning on push (helps in identifying software vulnerabilities, no additional charge)
```
Comment (eladb) [inline packages/aws-cdk/lib/api/toolkit-info.ts]: feels like this should be configurable, no?

Comment (jogold) [reply packages/aws-cdk/lib/api/toolkit-info.ts]: It adds security and is free, why?

Comment (eladb) [reply packages/aws-cdk/lib/api/toolkit-info.ts]: Let me ask it this way, could this block the push? If yes, we must allow users to opt-out.

Comment (jogold) [reply packages/aws-cdk/lib/api/toolkit-info.ts]: It's a asynchronous process and then 
> Amazon ECR sends an event to Amazon EventBridge (formerly called CloudWatch Events) when an image scan is completed.

Comment (eladb) [reply packages/aws-cdk/lib/api/toolkit-info.ts]: ok cool. so makes sense

**Comment by eladb** [comment]
@RomainMuller @rix0rrr @shivlaks any objections?

**Comment by jogold** [comment]
`Error: EISDIR: illegal operation on a directory, copyfile '/tmp/npm-packHNeqNF' -> '/tmp/npm-packNURlln/Amazon.CDK.CXAPI/npm-packHNeqNF'`??

**Comment by eladb** [comment]
Missed a beat for a second:

> Image Scanning for Amazon ECR is available at no additional charge, and you can now use it in all commercial AWS Regions and GovCloud (US)

**Comment by jogold** [comment]
Does it cover China regions?

**Comment by jogold** [comment]
> Does it cover China regions?

From the [docs](https://docs.amazonaws.cn/en_us/AmazonECR/latest/userguide/image-scanning.html), it seems that it's available in the China regions.

**Pull request merged**